<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wa-Tor SIMULATION</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="4" alignment="Qt::AlignTop">
     <widget class="QListWidget" name="palette">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="8" colspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>392</width>
         <height>499</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="nextchrononButton">
      <property name="text">
       <string>Next chronon</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" rowspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="age_fish">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Fish age:</string>
       </property>
       <property name="buddy">
        <cstring>fishBox</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="fishBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="age_shark">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>132</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Shark age:</string>
       </property>
       <property name="buddy">
        <cstring>sharkBox</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="sharkBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="energy_eat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Energy after eating:</string>
       </property>
       <property name="buddy">
        <cstring>eat_energyBox</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="eat_energyBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelpp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelpp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New simulation..</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load simulation..</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save simulation As..</string>
   </property>
  </action>
  <action name="QuitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNext_chronon">
   <property name="text">
    <string>Next chronon</string>
   </property>
  </action>
  <action name="actionSim">
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit Wa-Tor</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>QuitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
